contracts/KashiPair.sol

```diff
@@ -711,11 +711,11 @@ contract KashiPair is ERC20, BoringOwnable, IMasterContract {
             // solhint-disable-next-line reentrancy
             bentoBox.transfer(asset, address(this), masterContract.feeTo(), feeShare);
             totalAsset.elastic = totalAsset.elastic.add(returnedShare.sub(feeShare).to128());
-            emit LogAddAsset(address(swapper), address(this), extraShare.sub(feeShare), 0);
+            emit LogAddAsset(address(swapper), address(this), returnedShare.sub(feeShare), 0);
```

Comment:
We are already tracking the base amount through LogRepay, the former event was correct

Reply:
Thanks. I got this wrong because I was skipping the totalAsset check in a closed liquidation.
However, I have to find some way to detect this, otherwise the transfer check will be wrong

Reply:
Fixed. I ignore the bento transfer checks if `to == kashiPair` in `LogAddAsset`